cpp_linter/rest_api/github_api.py

```diff
@@ -457,6 +467,10 @@ def create_review_comments(
         :param review_comments: An object (passed by reference) that is used to store
             the results.
         """
+        if tidy_tool:
+            review_comments.tool_total["clang-tidy"] = 0
+        else:
+            review_comments.tool_total["clang-format"] = 0
```

Comment:
That's ok because it immediately precedes:
```py
    for file_obj in files:
```
In that loop we aggregate the `tool_total` data we need (and no where else).

See also `ReviewComments.__init__()` where we guarantee the key exists:
https://github.com/cpp-linter/cpp-linter/blob/4d67d414cbdf2406530e9846a4196488a70557ca/cpp_linter/clang_tools/patcher.py#L53-L56